testing/internal/e2e/tests/base_with_vault/auth_method_oidc_vault_test.go

```diff
@@ -312,4 +319,260 @@ func TestAuthMethodOidcVault(t *testing.T) {
+
+	// Boundary: retrieve the boundary auth token after a successful OIDC login
+	res, err = http.Post(
+		fmt.Sprintf("%s/v1/auth-methods/%s:authenticate", boundary.GetAddr(t), authMethodId),
+		"application/json",
+		strings.NewReader(
+			fmt.Sprintf(`{"command":"token", "attributes":{"token_id":"%s"}}`, oidcTokenId),
+		),
+	)
+	require.NoError(t, err)
+	t.Cleanup(func() {
+		res.Body.Close()
+	})
+	err = json.NewDecoder(res.Body).Decode(&authResult)
+	require.NoError(t, err)
+	require.Contains(t, authResult.Attributes, "token")
+	boundaryToken := authResult.Attributes["token"].(string)
+
+	// Try using the Boundary token to list scopes and users
+	t.Log("Using Boundary token...")
+	output = e2e.RunCommand(ctx, "boundary",
+		e2e.WithArgs(
+			"scopes",
+			"list",
+			"-token", "env://OIDC_USER_TOKEN",
+			"-format", "json",
+		),
+		e2e.WithEnv("OIDC_USER_TOKEN", boundaryToken),
+	)
+	require.NoError(t, output.Err, string(output.Stderr))
+
+	output = e2e.RunCommand(ctx, "boundary",
+		e2e.WithArgs(
+			"users",
+			"list",
+			"-token", "env://OIDC_USER_TOKEN",
+			"-format", "json",
+		),
+		e2e.WithEnv("OIDC_USER_TOKEN", boundaryToken),
+	)
+	require.Error(t, output.Err, string(output.Stderr))
+	var response boundary.CliError
+	err = json.Unmarshal(output.Stderr, &response)
+	require.NoError(t, err)
+	// User does not have permissions to list users
+	require.Equal(t, 403, response.Status)
+
+	// Do a user list without the token (using the admin login). Confirm that
+	// this operation works
+	output = e2e.RunCommand(ctx, "boundary",
+		e2e.WithArgs(
+			"users",
+			"list",
+			"-format", "json",
+		),
+	)
+	require.NoError(t, output.Err, string(output.Stderr))
+
+	// Validate account attributes
+	output = e2e.RunCommand(ctx, "boundary",
+		e2e.WithArgs(
+			"accounts",
+			"list",
+			"-auth-method-id", authMethodId,
+			"-format", "json",
+		),
+	)
+	require.NoError(t, output.Err, string(output.Stderr))
+	var accountListResult accounts.AccountListResult
+	err = json.Unmarshal(output.Stdout, &accountListResult)
+	require.NoError(t, err)
+	require.Len(t, accountListResult.Items, 1)
+	accountId := accountListResult.Items[0].Id
+
+	output = e2e.RunCommand(ctx, "boundary",
+		e2e.WithArgs(
+			"accounts",
+			"read",
+			"-id", accountId,
+			"-format", "json",
+		),
+	)
+	require.NoError(t, output.Err, string(output.Stderr))
+	var accountReadResult accounts.AccountReadResult
+	err = json.Unmarshal(output.Stdout, &accountReadResult)
+	require.NoError(t, err)
+	require.Contains(t, accountReadResult.Item.Attributes, "email")
+	require.Equal(t, userEmail, accountReadResult.Item.Attributes["email"])
+	// This field is set by the -account-claim-maps flag from above
+	require.Contains(t, accountReadResult.Item.Attributes, "full_name")
+	require.Equal(t, userName, accountReadResult.Item.Attributes["full_name"])
+
+	userInfoClaims, ok := accountReadResult.Item.Attributes["userinfo_claims"].(map[string]interface{})
```

Comment:
Nit: prefer `any` to `interface{}` in new code.
```suggestion
	userInfoClaims, ok := accountReadResult.Item.Attributes["userinfo_claims"].(map[string]any)
```